app/src/main/java/com/khuttun/notificationnotes/NotificationMgr.java

```diff
@@ -90,43 +85,158 @@ public void setGroupNotification(ArrayList<NotificationNote> notes)
                 break;
 
             default:
-                this.notificationManager.notify(GROUP_NOTIF_ID, createGroupNotification(visibleNotes));
+                if (groupAPI)
+                {
```

Comment:
It would be clearer to put the code for creating the grouped notifications to a function so that both if and else branches here could be one-liners.

Reply:
I've created the `displayGroupNotifications` function to do this.